crates/angstrom-net/src/manager.rs

```diff
@@ -12,16 +13,34 @@ use angstrom_types::{
     primitive::PeerId
 };
 use futures::StreamExt;
+use once_cell::unsync::Lazy;
 use reth_eth_wire::DisconnectReason;
 use reth_metrics::common::mpsc::UnboundedMeteredSender;
 use tokio::sync::mpsc::{UnboundedReceiver, UnboundedSender};
 use tokio_stream::wrappers::UnboundedReceiverStream;
 use tracing::error;
 
-use crate::{NetworkOrderEvent, StromMessage, StromNetworkHandleMsg, Swarm, SwarmEvent};
+use crate::{
+    CachedPeer, NetworkOrderEvent, StromMessage, StromNetworkHandleMsg, Swarm, SwarmEvent
+};
 #[allow(unused_imports)]
 use crate::{StromNetworkConfig, StromNetworkHandle, StromSessionManager};
 
+// use a thread local lazy to avoid synchronization overhead since path is
+// always the same
+thread_local! {
+    static CACHED_PEERS_TOML_PATH: Lazy<PathBuf> = Lazy::new(|| {
+        let mut path = PathBuf::new();
+        path.push(
+            homedir::my_home()
+                .unwrap()
+                .expect("Failed to get home directory. Please set the HOME environment variable.")
+        );
+        path.push(".angstrom_cached_peers.toml");
+        path
+    });
+}
```

Comment:
For better ux, should have some canonical location for all of the config files. If you look at the cli, we have arguments for node config + secret key location. Lets refactor that into a single configurable directory and then we can put all of that config and this in 1 location instead of having it scattered across there machine